Scripts/Events/EventDispatcher.cs

```diff
@@ -18,6 +18,12 @@ public class EventDispatcher : SingletonBehaviour<EventDispatcher>
         [SerializeField]
         private Dictionary<string, StandardEvent> eventMap = new Dictionary<string, StandardEvent>();
 
+        protected override void Awake()
+        {
+            _dontDestroyOnLoad = true;
```

Comment:
Per our face-to-face discussion, we will remove this line and instead set this via the inspector.